.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ jobs:
         os: [ubuntu-latest, windows-latest]
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
```

Comment:
Use the SHA corresponding to the version tag (here and later).

Reply:
I think I'm persuaded that for `actions/...` actions, it's better to effectively pinning to `2.*.*` as we do here, as I judge it more likely that we will thereby *get* security fixes, rather than the less likely event that github's `actions` account is compromised.